chrome/content/zotero/preferences/storageBreakdown.jsx

```diff
@@ -88,7 +88,10 @@ const StorageBreakdown = () => {
 
 						await Zotero.File.iterateDirectory(
 							Zotero.Attachments.getStorageDirectory(item).path,
-							getFileSize
+							(attachmentFile) => {
+								getFileSize(attachmentFile);
```

Comment:
Instead of ignoring the promise, probably better to add to an array and use `Promise.all()` after?